pkg/proxy/nftables/proxier.go

```diff
@@ -599,6 +599,11 @@ func (proxier *Proxier) setupNFTables(tx *knftables.Transaction) {
 		})
 	}
 
+	tx.Add(&knftables.Rule{
+		Chain: kubeEndpointsCheckChain,
+		Rule:  knftables.Concat("jump", kubeServiceEndpointsCheckChain),
+	})
```

Comment:
might be cleaner to have `service-endpoints-check` and `nodeport-endpoints-check` and have both linked to `filter-input` but only the former linked to `filter-forward` and `filter-output`?